src/components/MoneyRequestAmountInput.tsx

```diff
@@ -113,7 +125,7 @@ function MoneyRequestAmountInput(
     const textInput = useRef<BaseTextInputRef | null>(null);
 
     const decimals = CurrencyUtils.getCurrencyDecimals(currency);
-    const selectedAmountAsString = amount ? CurrencyUtils.convertToFrontendAmount(amount).toString() : '';
```

Comment:
I think it's been there because this is an old logic. selectedAmountAsString is used as the initial state for the amount and we don't use the correct formatting logic for our case.

Reply:
ah okay, that makes sense